.cirrus.yml

```diff
@@ -1,17 +1,3 @@
-task:
```

Comment:
it s tricky ... logically you re right but in the wild there are many users still taking time to migrate (in another project I had a freebsd ... 9.2 user, a bit extreme :) but you get me ..). I may suggest to give a bit of time before removing it but not up to me..

Reply:
9.2?  Wow.  My feeling is that while some users may take some time to upgrade, they also probably aren't using cutting edge Rust software.  So those users can probably make do with older crates.  No need to maintain the latest Rust releases for old FreeBSD release.  At least, that's my philosophy.

Reply:
Fair enough. It seems it takes more than this tough see [what it took for freebsd 11](https://github.com/rust-lang/homu/pull/169).

Reply:
Note that removing CI doesn't mean that the existing usage on FreeBSD 12 will be broken. The impact of this should be low, I think.